### PR TITLE
adding awesome.js

### PR DESCRIPTION
**Pull request by tdronkers04**

### CODE REVIEW

File: awesome.js [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log('GIT IS AWESOME!')
```
Comment (scogra17) [inline awesome.js]: No semicolon, eh? Too much time on Full Stack Open!

**Review by scogra17 — CHANGES_REQUESTED** [review]
It's good practice to add a blank line at the end of a file for unknown reasons. Please make this arbitrary change.

**Comment by katarinarosiak** [comment]
@tdronkers04  good job

**Review by scogra17 — APPROVED** [review]
It's good practice to add a blank line at the end of a file for unknown reasons. Please make this arbitrary change.

**Review by mabbason — APPROVED** [review]
Good w me too

**Pull request merged**